src/features/machines/getBalanceMachine.ts

```diff
@@ -133,3 +134,19 @@ export const getNearNep141StorageBalance = async ({
     throw new Error("Error fetching balance", { cause: err })
   }
 }
+
+export const getNearNep141MinStorageBalance = async ({
+  contractId,
+}: {
+  contractId: string
+}): Promise<bigint> => {
+  const response = await getNearNep141StorageBalanceBounds({
+    request_type: "call_function",
+    method_name: "storage_balance_bounds",
+    account_id: contractId,
+    args_base64: "e30=",
```

Comment:
What is this magic? :) Let's write it explicity `base64.encode(JSON.stringify({}))`